configure.acr

```diff
@@ -34,6 +34,8 @@ IFAND WITH_CAPSTONE HAVE_PKGCFG_CAPSTONE {
 	USE_CAPSTONE = 1 ;
 }{
 	USE_CAPSTONE = 0 ;
+	CAPSTONE_CFLAGS = '' ;
```

Comment:
shouldnt this be done by acr itself? in the PKGCFG directive? it will override user defined env vars, but at least it will be consistent

Reply:
i have no idea :) i don't know acr, but i'm not sure how PKGCFG could do this. The problem is: you have capstone installed, but you still want to use embedded capstone.

I *think* this means that:
1) PKGCFG finds that capstone is installed, so it sets CAPSTONE_CFLAGS and CAPSTONE_LDFLAGS
2) WITH_CAPSTONE is false, so it goes in the else case and it sets USE_CAPSTONE=0, but still, CAPSTONE_CFLAGS and CAPSTONE_LDFLAGS have been already set.